doc/api/stream.md

```diff
@@ -347,6 +347,8 @@ buffer that would have an adverse impact on performance. In such situations,
 implementations that implement the `writable._writev()` method can perform
 buffered writes in a more optimized manner.
 
+See also: [`writable.uncork()`][].
```

Comment:
Does this work without defining `cork()` and `uncork()` in the links at the bottom of the page?